packages/core/ui-settings/core-ui-settings-server-internal/src/saved_objects/ui_settings.ts

```diff
@@ -43,3 +43,29 @@ export const uiSettingsType: SavedObjectsType = {
   },
   migrations,
 };
+
+export const uiSettingsGlobalType: SavedObjectsType = {
+  name: 'config-global',
+  hidden: true,
+  namespaceType: 'agnostic',
+  mappings: {
+    dynamic: false,
+    properties: {
+      buildNum: {
+        type: 'keyword',
+      },
+    },
+  },
+  management: {
+    importableAndExportable: true,
+    getInAppUrl() {
+      return {
+        path: `/app/management/kibana/settings`,
+        uiCapabilitiesPath: 'advancedSettings.show',
+      };
+    },
+    getTitle(obj) {
+      return `Global Advanced Settings [${obj.id}]`;
```

Comment:
nit: WDYT about not calling them "advanced"?

```suggestion
      return `Global Setting [${obj.id}]`;
```